compose/src/main/java/com/bottlerocketstudios/compose/snippets/snippetDetails/snippetDetailsWidgets/SnippetDetailsPrimaryButton.kt

```diff
@@ -0,0 +1,61 @@
+package com.bottlerocketstudios.compose.snippets.snippetDetails.snippetDetailsWidgets
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import com.bottlerocketstudios.compose.resources.Colors
+import com.bottlerocketstudios.compose.resources.Dimens
+import com.bottlerocketstudios.compose.resources.typography
+import com.bottlerocketstudios.compose.util.Preview
+
+@Composable
+fun SnippetDetailsPrimaryButton(
```

Comment:
This could probably be name SnippetDetailsButton so as not to confuse it with the existing primary button composable